README.md

```diff
@@ -28,7 +28,7 @@ jobs:
     uses: nrkno/github-workflow-semantic-release/.github/workflows/workflow.yaml@v1
     with:
       # Define the type of machine to run the jobs on.
-      # Default: self-hosted
+      # Default: nrk-azure-intern
```

Comment:
As this is a public repo, could we do [ self-hosted, linux] instead?

Reply:
We are unable to specify a list of values. Strings only, and I'm struggeling to find a way to convert string values to lists that are accepted as lists to `runs-on` in the workflow itself :(

Reply:
Passing the list to the input as `runs-on: "[ self-hosted, linux ]"` would work, no?

Reply:
Also, the default is already set to `nrk-azure-intern`, this PR only updates the doc to reflect the actual default.

Reply:
> Passing the list to the input as `runs-on: "[ self-hosted, linux ]"` would work, no?

No, because it is passed as an argument to the workflow and that variable is defined as a `string`. This is not the _native_ runs-on argument.